service/build.gradle

```diff
@@ -53,6 +53,18 @@ dependencies {
 	}
 	// https://mvnrepository.com/artifact/org.mockito/mockito-core
 	testImplementation group: 'org.mockito', name: 'mockito-core', version: '4.4.0'
+
+	// OpenTelemetry
+	var openTelemetryVersion = '1.31.0'
```

Comment:
After seeing this in different PRs I'm wondering if we should expose this dependency from tcl, with the idea that all terra services should be using tracing and opentelemetry.

Reply:
I'm glad you brought this up, I was wondering about this too.  @dvoet , is there a reason why this block isn't handled by default in TCL?